src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -249,8 +249,6 @@ public Future<Void> updateOrder(CompositePurchaseOrder compPO, boolean deleteHol
           .compose(v -> {
             if (isTransitionToOpen) {
               compPO.getPoLines().forEach(poLine -> PoLineCommonUtil.updateLocationsQuantity(poLine.getLocations()));
-              return openCompositeOrderFlowValidator.checkLocationsAndPiecesConsistency(compPO.getPoLines(), requestContext)
-                .compose(ok -> openCompositeOrderFlowValidator.checkFundLocationRestrictions(compPO.getPoLines(), requestContext));
```

Comment:
duplicate logic, it exists already  in
https://github.com/folio-org/mod-orders/pull/1161/files#diff-1817ae44495cc630f4866f6032c4312fe9429d1e1c35ac558dd4ae43641774bdR85